modules/sdk-coin-avaxp/src/lib/transaction.ts

```diff
@@ -68,6 +68,7 @@ export class Transaction extends BaseTransaction {
   public _fromAddresses: BufferAvax[] = [];
   public _rewardAddresses: BufferAvax[];
   public _utxos: DecodedUtxoObj[] = [];
+  public _fee: BN = new BN(0);
```

Comment:
fee is not 0 in terms of export and import

Reply:
That is the default fee for AddValidator and AddDelegator txs. 
Should I remove default value and set fee in AddDelegator?